src/explorer.api/Models/ExploreResult.cs

```diff
@@ -0,0 +1,47 @@
+namespace Explorer
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json.Serialization;
+    using Aircloak.JsonApi.ResponseTypes;
+
+    internal class ExploreResult
+    {
+        public ExploreResult(Guid explorationId, string status)
+        {
+            Id = explorationId;
+            Status = status;
+            Metrics = new List<Metric>();
```

Comment:
In order to avoid allocation of an object which will get unused, I think that we can use this instead,: `Metrics = Array.Empty<Metric>();`

Reply:
Thanks for the tip, I didn't know this!